modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -4,15 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, raise, error
 
-This Core component generates a xref:mule-error-concept.adoc[Mule Error], as if a failure had occurred, which allows you to customize its description and type.
+This Core component generates a Mule Error, as if a failure had occurred, which allows you to customize its description and type.
 
-Note that you can only use core types, such as `SECURITY` or custom ones. You cannot raise an
-error from another module like `FILE:FILE_NOT_FOUND`.
+Use this component to only raise core runtime errors, such as `MULE:SECURITY`, `MULE:CONNECTIVITY`, etc, or custom ones.
+You cannot raise an error from another connector or module like `FILE:FILE_NOT_FOUND`, `JSON:INVALID_SCHEMA`. In fact, you can not use their existing namespace.
```

Comment:
Consistency: use "cannot" or "can not" in all cases. 
Remove "In fact, ..." > "`JSON:INVALID_SCHEMA`. You cannot use a connector's existing namespace"